internal/cli/deployment_list.go

```diff
@@ -112,22 +108,14 @@ func (c *DeploymentListCommand) Run(args []string) int {
 		sort.Sort(serversort.DeploymentCompleteDesc(resp.Deployments))
 
 		// get status reports
-		statusReportResp, err := client.GetLatestStatusReport(ctx, &pb.GetLatestStatusReportRequest{
+		statusReportsResp, err := client.ListStatusReports(ctx, &pb.ListStatusReportsRequest{
 			Application: app.Ref(),
 			Workspace:   wsRef,
 		})
+
 		if status.Code(err) == codes.NotFound {
```

Comment:
Might need to rebase to get Raes fix for when the API for status reports is unimplemented here.